includes/class-wc-gateway-stripe.php

```diff
@@ -336,10 +347,48 @@ public function elements_form() {
 		<?php
 	}
 
+	/**
+	 * Returns the URL of the configuration screen for this gateway, for use in internal links.
+	 *
+	 * @return string URL of the configuration screen for this gateway
+	 */
+	public static function get_settings_url() {
+		return admin_url( add_query_arg( self::$settings_url_params, 'admin.php' ) );
+	}
+
+	/**
+	 * Output the React Settings page wrapper
+	 */
+	private function output_settings_page_wrapper() {
+		global $hide_save_button;
+		$hide_save_button = true;
+		?>
+		<div>
+			<ul class="subsubsub">
+				<li><a href="<?php echo esc_html( self::get_settings_url() ); ?>" class="current"><?php echo esc_html( $this->get_method_title() ); ?></a></li>
+				<li>|</li>
+				<li><a href="<?php echo esc_html( admin_url( 'admin.php?page=wc-settings&tab=checkout' ) ); ?>"><?php echo esc_html( __( 'All payment methods', 'woocommerce-gateway-stripe' ) ); ?></a></li>
+			</ul>
+		</div>
```

Comment:
I don't see this as part of the designs. But if needed, WC provides a hook for it: https://github.com/Automattic/woocommerce-payments/blob/d8ba70898ddbd7169cb48fc426fb80d82a61b1d3/includes/admin/class-wc-payments-admin-sections-overwrite.php#L64-L72

Reply:
Thanks for pointing that out. You're right; Figma shows this:

![Screen Shot 2021-08-19 at 4 17 47 PM](https://user-images.githubusercontent.com/6723003/130084863-7fe5ccb9-4944-4bc6-8185-c71455281e4d.png)

I personally like this which comes from core:
![Screen Shot 2021-08-19 at 4 19 17 PM](https://user-images.githubusercontent.com/6723003/130085071-88a96aa9-bafc-4454-87d6-d43fe99a0d24.png)

I think I favor the core WC method of providing a link back to "all payment methods", otherwise, you'd have to click on "payments" which is a tab that is already open. Seems sus from a UX perspective, no? 🤔 

I'll loop in design.

Reply:
The "Payments" link in the breadcrumb links to the "all payment methods" section.

What do you think about removing what you have here and writing a separate ticket for eventual adjustments, so that we can start getting the rest of the layout merged?

Reply:
Done. Ready for another look 👍